clusterous/cluster.py

```diff
@@ -1127,44 +1196,74 @@ def add_nodes(self, num_nodes, instance_type, node_name):
         vpc = self._get_vpc(vpc_conn)
         private_subnet = self._create_subnet(vpc_conn, vpc, 'private-subnet')
         private_security_group = self._create_private_sg(vpc_conn, vpc, "private-sg")
+        node_block_devices = boto.ec2.blockdevicemapping.BlockDeviceMapping(conn)
+        node_root_vol = boto.ec2.blockdevicemapping.BlockDeviceType(connection=conn, delete_on_termination=True, volume_type='gp2')
+        node_block_devices['/dev/sda1'] = node_root_vol
```

Comment:
This is a good idea, but it needs to be set for regular nodes as well spot nodes